js/utils/Logger/Logger.ts

```diff
@@ -2,36 +2,28 @@ module hirespace {
     'use strict';
 
     export class Logger {
-        static log(data: any) {
-            if (hirespace.Debug.getEnvironment() !== 'development') {
-                return false;
-            }
+        static execute(type: string, data: any, notify?: boolean) {
```

Comment:
If you need to unit-test it, then you can extract it to the external utils class, and test it through it. In such way you will have a clean Logger class, and also testable "private method"